src/manifolds/Euclidean.jl

```diff
@@ -280,13 +274,21 @@ inner(::Euclidean, ::Any...)
 @inline inner(::Euclidean, p, X, Y) = dot(X, Y)
 @inline inner(::MetricManifold{𝔽,<:Manifold,EuclideanMetric}, p, X, Y) where {𝔽} = dot(X, Y)
 
-function inverse_local_metric(M::MetricManifold{𝔽,<:Manifold,EuclideanMetric}, p) where {𝔽}
-    return local_metric(M, p)
+function inverse_local_metric(
+    M::MetricManifold{𝔽,<:Manifold,EuclideanMetric},
+    B::InducedBasis{𝔽,TangentSpaceType,<:RetractionAtlas},
+    p,
+) where {𝔽}
+    return local_metric(M, B, p)
```

Comment:
I know this is an old code line, but I am confused that the inverse of the local metric actually calls the local metric?

Reply:
Well, both should just be unit matrices in this basis I think?

Reply:
I am not 100% sure.

Reply:
It should be the identity matrix when the basis is orthonormal. If the basis is only orthogonal, then the local metric is a diagonal matrix, not the identity matrix, and it is not equal to its inverse.

Reply:
Yes, right, so I think I'll just restrict these two methods for `inverse_local_metric` and `local_metric` to orthonormal bases (in the retraction atlas).

Reply:
I think for that we need an orthonormality trait for bases so that we don't have to special-case bases. `InducedBasis` really increases the problem of determining whether a given basis is orthonormal or not.